.github/workflows/chart-lint-test.yml

```diff
@@ -0,0 +1,50 @@
+name: Chart Lint and Test
+
+on:
+  push:
+    paths:
+      - 'deploy/helm/**'
+    branches:
+      - develop
+  pull_request:
+    paths:
+      - 'deploy/helm/**'
+    branches:
+      - develop
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --config ct.yaml)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml
```

Comment:
Please include a file like this (https://github.com/openebs/zfs-localpv/blob/master/ct.yaml) in the root directory of this repo.

Reply:
Added